crates/dbsp/src/circuit/runtime.rs

```diff
@@ -404,24 +412,21 @@ impl Runtime {
         BACKEND.with(|rc| rc.clone())
     }
 
-    /// Returns this thread's buffer cache.
+    /// Returns this thread's buffer cache, if storage is configured.
     ///
     /// The buffer cache is shared between a worker and its background thread,
     /// because they access the same files. They are otherwise independent,
     /// because different workers access different files.
-    pub fn buffer_cache() -> Arc<BufferCache<FileCacheEntry>> {
-        if let Some(rt) = Runtime::runtime() {
-            let cache = rt
-                .local_store()
-                .get(&BufferCacheId(Runtime::worker_index()));
-            cache.unwrap().clone()
-        } else {
-            // No `Runtime` means there's only a single worker, so use a single
```

Comment:
did we get rid of the tests that dont have  a runtime?

Reply:
A `Runtime` is the only way to configure storage, and without a Runtime, Runtime::min_storage_bytes() always returns None, which tells FallbackIndexedWSet and friends not to spill to storage, so without a runtime we'll never try to use storage.